### DISCUSSION TITLE
Shaders

### DISCUSSION THREAD

**Discussion by randomaccount20** [DEMOLITIONDON96/Demolition-Engine#79] [General (Friday Night Chattin')]
So hi, i just want to say that i can't find the shaders page, anyone who knows the link for that?, And also, how do i access debug mode?

**Comment by ThatOneSillyGuyDon** [comment]
Check the wiki under the Lua Contents page

https://github.com/DEMOLITIONDON96/Demolition-Engine/wiki/New-Lua-Content

**Comment by charlesisfeline** [comment]
1. https://github.com/DEMOLITIONDON96/Demolition-Engine/wiki/New-Lua-Content
2. press 7 in any song

**Reply by ThatOneSillyGuyDon** [reply]
And the Debug Mode option in the settings is kinda pointless atm, for all it does is allow you to press 1 to skip songs

**Reply by randomaccount20** [reply]
i thought it was a actual debug mode thats just chart editor
